src/Rokt-Kit.js

```diff
@@ -187,6 +198,40 @@ var constructor = function () {
     this.selectPlacements = selectPlacements;
 
     // mParticle Kit Callback Methods
+    function fetchOptimizely() {
+        var forwarders = window.mParticle
+            ._getActiveForwarders()
+            .filter(function (forwarder) {
+                return forwarder.name === 'Optimizely';
+            });
+
+        try {
+            if (forwarders.length > 0 || window.optimizely) {
```

Comment:
I think logical AND would be more appropriate, right?

```suggestion
            if (forwarders.length > 0 && window.optimizely) {
```

Reply:
but what about the case where the optimizely sdk wasn't loaded via the kit setup, since the client can still load it directly?

[mparticle doc on optimizely integration](https://docs.mparticle.com/integrations/optimizely/event/#web)
> While allowing mParticle to automatic load Optimizely reduces the amount of code you need to write, you may choose to initialize Optimizely yourself to prevent “page flashing” in the case where an Optimizely experiment is expected to alter the UI immediately on load. You can [read more about this concern here](https://help.optimizely.com/Set_Up_Optimizely/Implement_the_one-line_snippet_for_Optimizely_X) and make the choice that’s best for your setup.

i guess even if they load it themselves, we'd still have a forwarder?

Reply:
forwarder would still exist yes.

Reply:
ok got it. in that case yeah, i'll modify this to &&